src/layer/image.ts

```diff
@@ -1,7 +1,7 @@
-import { Visual, VisualOptions } from './visual'
+import { Visual, VisualSourceOptions} from './visual'
 import { VisualSourceMixin } from './visual-source'
```

Comment:
You can import VisualSourceOptions here

Reply:
Yeah thanks a lot I have made the necessary changes and made a new PR.

Thanks for the suggestion again this was my first Open source contribution and I am a complete newbie.

Reply:
No worries! One more thing, run `npm run lint && npm run build && npm run test`. Looks like there are a few linter warnings. Also, npm run build will update the dist folder, which needs to be committed too.

Reply:
Actually the issue is I am getting node gyp errors on l `npm install` plus I am getting this error on running ` npm run lint && npm run build && npm run test` and I don't know how to resolve any of these.
![Screenshot (8490)](https://user-images.githubusercontent.com/96020697/155014882-d5bc351c-7397-426e-acc3-762e3a05839c.png)

Reply:
I am sorry but after trying for quite some time now I have not really found an optimal fix to this

Reply:
Thanks for looking into it, try running this if you haven't already (more info can be found [here](https://stackoverflow.com/questions/35293117/npm-install-that-requires-node-gyp-fails-on-windows)):
```sh
npm install -g windows-build-tools
```

Reply:
I actually did try just that however I am getting another installation error as maybe because the [package has since been deprecated ](https://www.npmjs.com/package/windows-build-tools)

I also have visual studio desktop developer installed however these errors still persist

Reply:
If linting still doesn't work, please send the output of `npm list eslint`, because I think you have the wrong version of eslint.

Reply:
yeah the folder is empty which is weird cause I am pretty sure I did install eslint
![Screenshot (8493)](https://user-images.githubusercontent.com/96020697/155035853-51e25c80-53f1-415d-885b-bc001186c8bd.png)

Reply:
Is there any fix to this ?

Reply:
Which version of visual studio do you have?

Reply:
latest. Also the buildkit contains the something something development with C++.

Reply:
It turns out upgrading resemblejs fixed the issue for me when testing this issue with windows. If you rebase onto the latest branch it should work.

I just renamed vidar to etro, because I found out about a virus named vidar.. so if there are a lot of merge conflicts I can do the rebase myself using a renaming script.